.gitignore

```diff
@@ -5,3 +5,4 @@ _site/**
 .sass-cache/**
 CNAME
 .travis.yml
+.idea/
```

Comment:
Please push that change in another commit. Also please squash this change into a single commit. Thanks.